drivers/serial/Kconfig

```diff
@@ -31,6 +31,12 @@ config SERIAL_SUPPORT_INTERRUPT
 	  This is an option to be enabled by individual serial driver
 	  to signal that the driver and hardware supports interrupts.
 
+config SERIAL_SUPPORT_RS_485
+	bool
+	help
+	  This is an option to be enabled by individual serival driver
+	  to signal that the driver and hardware supports RS_455 interface.
```

Comment:
typo /RS_455/RS_485

Reply:
Thanks for pointing  out the typo.  Fixed in the latest revision